swayidle/meson.build

```diff
@@ -0,0 +1,13 @@
+executable(
+	'swayidle', [
+		'main.c',
+	],
+	include_directories: [sway_inc],
+	dependencies: [
+		client_protos,
+		wayland_client,
+		wlroots,
+	],
+	link_with: [lib_sway_common, lib_sway_client],
+    install: true
```

Comment:
You could grep for two spaces, or configure your editor to show invisible chars, or configure your editor to use the [`.editorconfig`](http://editorconfig.org/) file.

Reply:
This comment also indicates spaces instead of tabs.